### DISCUSSION TITLE
doris实例中/mnt/data/1 的磁盘水位会突然暴涨,导致一些查询查询不了,我该怎么排查?

### DISCUSSION THREAD

**Discussion by shine10076** [apache/doris#20157] [A - General / Q&A]
doris实例中/mnt/data/1 的磁盘水位会突然暴涨,导致一些查询查询不了,我该怎么排查?

**Comment by LemonLiTree** [comment]
What is wrong with the query?

**Reply by shine10076** [reply]
detailMessage = there is no scanNode Backend. [11002: not alive]

**Reply by LemonLiTree** [reply]
> detailMessage = there is no scanNode Backend. [11002: not alive]

be not alive, restart ，If the restart fails, see what is wrong
